image-layout.md

```diff
@@ -149,6 +149,7 @@ This index provides an established path (`/index.json`) to have an entry point f
 * Future versions of the spec MAY use a different mediatype (i.e. a new versioned format).
 * An encountered `mediaType` that is unknown SHOULD be safely ignored.
 
+Note that while `index.json` is a REQUIRED file, the use of an image index is OPTIONAL for image providers.
```

Comment:
I'd shift this line up to the [opening section][1] and rephrase it to something like:

> Consumers MAY support OCI Image Layout and MAY also support other mechanisms for distributing image content.

The optionality of image-layout support is important enough that I'd rather not bury it 150 lines into the file ;).

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0/image-layout.md#L1-L10